.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.biome": "explicit",
+    "source.organizeImports.biome": "explicit"
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "biomejs.biome" 
+  },
+  "[jsonc]": {
+    "editor.defaultFormatter": "biomejs.biome" 
+  }
```

Comment:
would be nice to include `"openInGitHub.defaultBranch": "unstable"` for [Open In GitHub](https://marketplace.visualstudio.com/items?itemName=sysoev.vscode-open-in-github) extension